Events.php

```diff
@@ -110,8 +110,10 @@ public static function onBeforeControllerAction(ActionEvent $event)
 
         // Allow legal module usage
         if ($event->action->controller->module->id === 'legal') {
```

Comment:
@funkycram Thanks!

Would that also be correct in this situation?
`($event->action->controller->module->id === 'legal' && $event->action->controller->id === 'page')`

Reply:
I think it's better to keep the `return;` for all cases matching `if ($event->action->controller->module->id === 'legal')` otherwise admin will not be able to access to `AdminController` if `hasOpenCheck()` is true for them.

Reply:
@funkycram Ok got it! Can you please adapt the CHANGELOG.md with this change? Then we can merge it! Thanks!

Reply:
Done: https://github.com/humhub-contrib/legal/pull/28